integration/stability/hypervisor_stability_kill_test.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 HyperHQ Inc.
```

Comment:
OOI, Is the company name still correct?

Reply:
We are still using hyper.sh for kata containers development so far ;)

Reply:
ack ;)